src/main/java/org/springframework/data/elasticsearch/repository/support/MappingElasticsearchEntityInformation.java

```diff
@@ -47,9 +47,6 @@ public MappingElasticsearchEntityInformation(ElasticsearchPersistentEntity<T> en
 	public MappingElasticsearchEntityInformation(ElasticsearchPersistentEntity<T> entity, String indexName, String type) {
```

Comment:
Please add yourself as `@author` in the class comment

Reply:
done